Makefile

```diff
@@ -1,14 +1,15 @@
 ###########################
 # Configuration Variables #
 ###########################
-ORG := github.com/operator-framework
-PKG := $(ORG)/combo
+PKG := $(shell head -n 1 go.mod | awk '{print $$2}')
```

Comment:
I don't think it's worth parsing the go.mod to pull out the repository package. If we had to change the repository name, it's not the end of the world having to change any hardcoded assumptions throughout the codebase in multiple places.

Reply:
@njhale do you have an opinion on this particular piece?